internal/libs/autofile/autofile.go

```
@@ -83,33 +84,68 @@ func OpenAutoFile(path string) (*AutoFile, error) {
 		}
 	}()
 
-	go af.closeFileRoutine()
+	go af.syncFileRoutine()
 
 	return af, nil
 }
 
 // Close shuts down the closing goroutine, SIGHUP handler and closes the
 // AutoFile.
 func (af *AutoFile) Close() error {
-	af.closeTicker.Stop()
+	af.refreshTicker.Stop()
 	close(af.closeTickerStopc)
 	if af.hupc != nil {
 		close(af.hupc)
 	}
 	return af.closeFile()
 }
 
-func (af *AutoFile) closeFileRoutine() {
+func (af *AutoFile) syncFileRoutine() {
 	for {
 		select {
-		case <-af.closeTicker.C:
-			_ = af.closeFile()
+		case <-af.refreshTicker.C:
+			_ = af.syncFile()
```

why do we ignore this error?

---

We didn't do anything with it before, this change mimicks prior code.